examples/basic/main.tf

```diff
@@ -7,6 +7,6 @@ provider "helm" {
 module "ocean-network-client" {
   source            = "../.."
   image_pull_policy = "Always"
-  image_tag         = "1.0.0"
+  image_tag         = "1.1.0"
```

Comment:
This is the image tag?

Reply:
no, my bad, mixed up the helm version and the image version